dojo/tools/api_sonarqube/importer.py

```diff
@@ -141,17 +143,22 @@ def import_issues(self, test):
                 rule_id = issue["rule"]
                 rule = client.get_rule(rule_id)
                 severity = self.convert_sonar_severity(issue["severity"])
+                try:
+                    sonarqube_permalink = f"[Issue permalink]({sonarUrl}project/issues?issues={issue['key']}&open={issue['key']}&resolved={issue['status']}&id={issue['project']}) \n"
```

Comment:
What is the purpose of `open` and `resolved` part?
It looks that work also without them.
I do not even know where to click to have `open` in url. How did you get it? I would like to learn something new today :)

Reply:
The required URL structure is really weird. I gave up to understand it to be honest. 

The problem is that if you only enter the ID within the url as when you click around in the UI, sonarqube sometimes throws an error called "the finding can not be displayed as it was not part of the 1000 initial loaded". However if you try to open the finding with the other parameters it works. If you want to share an Issue via the UI, klick on the "get permalink" option and you will receive a similar URL. 

I got the exact solution from [here](https://community.sonarsource.com/t/constructing-issue-link-from-sonarqube-api-response/54719/3)

Reply:
I also tried it with less GET parameters but sometimes got the same error so I will leave them in as the data is collected from the API anyway...

Reply:
Ok, in that case, it is strong argument to say: keep it as it is.
Thank for searching it and posting link to SQ forum.